recipes/pyopcode/bld.bat

```diff
@@ -0,0 +1,29 @@
+set BUILD_CONFIG=Release
+REM tell cmake where Python is
+set PYTHON_LIBRARY=%PREFIX%\libs\python%PY_VER:~0,1%%PY_VER:~2,1%.lib
+
+REM work in build subdir
+cd pyopcode
+mkdir build
+cd build
+
+cmake ../src -G%CMAKE_GENERATOR% ^
+    -Wno-dev ^
```

Comment:
I think this flag might be [confusing]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.3328/job/9fmtgpv3qibeakgg#L330 ) it.